libs/db/src/lib/schemas/annotations.schema.ts

```diff
@@ -0,0 +1,95 @@
+import { IAnnotations } from '@dua-upd/types-common';
+import { Prop, Schema, SchemaFactory } from '@nestjs/mongoose';
+import { Document, model, Types } from 'mongoose';
+import { Task } from './task.schema';
+
+export type AnnotationsDocument = Annotations & Document;
+
+@Schema()
+export class Annotations implements IAnnotations {
+  @Prop({ type: Types.ObjectId, required: true })
+  _id: Types.ObjectId = new Types.ObjectId();
+
+  @Prop({ type: String, required: true, unique: true, index: true })
+  airtable_id = '';
+
+  @Prop({ type: String, required: true })
+  title: string;
+
+  @Prop({ type: String, required: true })
+  title_fr: string;
+
+  @Prop({ type: String })
+  event_type:
```

Comment:
I would create a type alias for this because it's used in multiple places and we want to avoid having to change it in multiple places because that's really error prone. And it makes it a bit shorter which is always a plus.